Formula/gq-cli.rb

```diff
@@ -0,0 +1,20 @@
+class GqCli < Formula
+  desc "Command-line interface for GQ"
+  homepage "https://goquant.io"
+  url "https://brickelldatabros1aboah.blob.core.windows.net/public/gq-cli-source-0.0.4.tar.gz"
+  sha256 "6e2b86cf57035bba045d7db9f1d28ae83283cec6b0ed4335083f320374b951ba"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root=#{prefix}", "--path=.", "--debug"
```

Comment:
Please don't use a debug profile.

```suggestion
    system "cargo", "install", *std_cargo_args
```

Reply:
Debug in this case will cause a break in the code when attempting to access the live resource. The source code can be changed to circumvent this, but our inital testers won't have as much information. And because we are a small team there is only so much we can do without external help.